packages/dataclass-editor/src/master/ValidationRow.tsx

```diff
@@ -0,0 +1,42 @@
+import { MessageRow, SelectRow, TableCell } from '@axonivy/ui-components';
+import { flexRender, type Row } from '@tanstack/react-table';
+import { useAppContext } from '../context/AppContext';
+import type { DataClassField } from '../data/dataclass';
+import type { ValidationMessage } from '../protocol/types';
+import './ValidationRow.css';
+
+export const validationMessagesOfRow = (validationMessages: Array<ValidationMessage>, field: DataClassField) => {
+  return validationMessages.filter(message => message.path === field.name);
+};
```

Comment:
I suggest that also here you build an custom hook where you can get your validations for a specific path. Somethinkg like we've in the inscription client: https://github.com/axonivy/inscription-client/blob/master/packages/editor/src/context/useValidation.ts